codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/integration/HttpProtocolUnitTestRequestGenerator.java

```diff
@@ -282,7 +274,8 @@ protected void generateTestServer(
             String name,
             Consumer<GoWriter> handler
     ) {
-        super.generateTestServer(writer, name, handler);
+        // We aren't using a test server, but we do need a URL to set.
+        writer.write("serverURL := \"http://localhost:8888/\"");
```

Comment:
Can't we just remove this entire `generateTestServer` method? Is its code needed at all anymore?

Reply:
If we do that we have to remove it from super class `HttpProtocolUnitTestGenerator` and simply provide this fake url to request and response generator separately.

Reply:
Also we need the test cases' host info in some host test cases, I prefer to keep it now in case of breaking changes.